src/cli/argument-parser/index.ts

```diff
@@ -150,66 +148,66 @@ export default class CLIArgumentParser {
+            .option('-a, --app <command>', 'execute a shell command on startup to launch a web application or perform other preparatory tasks')
             .option('-c, --concurrency <number>', 'run tests concurrently')
-            .option('-L, --live', 'enable live mode. In this mode, TestCafe watches for changes you make in the test files. These changes immediately restart the tests so that you can see the effect.')
-            .option('--test-meta <key=value[,key2=value2,...]>', 'run only tests with matching metadata')
-            .option('--fixture-meta <key=value[,key2=value2,...]>', 'run only fixtures with matching metadata')
-            .option('--debug-on-fail', 'pause the test if it fails')
-            .option('--app-init-delay <ms>', 'specify how much time it takes for the tested app to initialize')
-            .option('--selector-timeout <ms>', 'specify the time within which selectors make attempts to obtain a node to be returned')
-            .option('--assertion-timeout <ms>', 'specify the time within which assertion should pass')
-            .option('--page-load-timeout <ms>', 'specify the time within which TestCafe waits for the `window.load` event to fire on page load before proceeding to the next test action')
-            .option('--page-request-timeout <ms>', "specifies the timeout in milliseconds to complete the request for the page's HTML")
-            .option('--ajax-request-timeout <ms>', 'specifies the timeout in milliseconds to complete the AJAX requests (XHR or fetch)')
-            .option('--browser-init-timeout <ms>', 'specify the time (in milliseconds) TestCafe waits for the browser to start')
-            .option('--test-execution-timeout <ms>', 'specify the time (in milliseconds) TestCafe waits for the test executed')
-            .option('--run-execution-timeout <ms>', 'specify the time (in milliseconds) TestCafe waits for the all test executed')
-            .option('--speed <factor>', 'set the speed of test execution (0.01 ... 1)')
-            .option('--ports <port1,port2>', 'specify custom port numbers')
-            .option('--hostname <name>', 'specify the hostname')
-            .option('--proxy <host>', 'specify the host of the proxy server')
-            .option('--proxy-bypass <rules>', 'specify a comma-separated list of rules that define URLs accessed bypassing the proxy server')
-            .option('--ssl <options>', 'specify SSL options to run TestCafe proxy server over the HTTPS protocol')
+            .option('-L, --live', 'enable live mode. Live mode restarts tests when you make changes to test files.')
+            .option('--test-meta <key=value[,key2=value2,...]>', 'filter tests by metadata')
+            .option('--fixture-meta <key=value[,key2=value2,...]>', 'filter fixtures by metadata')
+            .option('--debug-on-fail', 'pause tests on failure')
+            .option('--experimental-proxyless', 'automate Google Chrome with the CDP protocol')
```

Comment:
In the regular way (non proxyless), we use the CDP API to perform some actions - record video, take screenshot, emulate devices, etc. E.g., in the regular way we automate Google Chrome using the CDP API.
The main idea: we don't use the url-rewritten proxy to inject the testcafe stuff into the tested page.
I propose the following variant of the option description: `run tests in proxyless mode (see more  information in <url>)`.
The <url> describes what is the 'proxyless mode', which benefits it provides and so on.

Reply:
```suggestion
            .option('--experimental-proxyless', 'enable proxyless mode: https://testcafe.io/documentation/404237/guides/experimental-capabilities/proxyless-mode')
```